tools/text_denormalization/classify/tokenize_and_classify.grm

```diff
@@ -16,7 +16,7 @@
 import 'punctuation.grm' as p;
 import '../util.grm' as u;
 
-types = LoadFstFromFar['tokenize_and_classify_classify.far', 'TOKENIZE_AND_CLASSIFY'];
+types = LoadFstFromFar['tokenize_and_classify_tmp.far', 'TOKENIZE_AND_CLASSIFY'];
```

Comment:
Let's change to `tokenize_and_classify.far`

Reply:
keeping as it is as discussed